packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -473,6 +478,123 @@ const main = async () => {
             )
         })
 
+    program
```

Comment:
Lets move the scripts into the extension sdk project so everything is in one cohesive place with regards to extensibility. We should however add commands to this file that will ultimately run the binary that gets added from the package dependency.

Reply:
Done, I moved the script to packages/pwa-kit-extension-sdk/bin/list-overrides.js